### PR TITLE
NXP-29504: adapt usage of runtime messages api

### PR DESCRIPTION
**Pull request by atchertchian**
Will need a nuxeo master successful deployment for build to go through

### CODE REVIEW

**Comment by atchertchian** [comment]
Forgot an api usage -> will force push to include it

File: modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/snapshot/SnapshotManagerComponent.java [hunk 1]
```diff
@@ -420,7 +421,7 @@ public void start(ComponentContext context) {
                         "Failed to register contribution with id '%s' on '%s': error initializing class '%s' (%s).",
                         descriptor.getId(), name, descriptor.getKlass(), e.toString());
                 log.error(msg, e);
-                Framework.getRuntime().getMessageHandler().addError(msg);
+                Framework.getRuntime().getMessageHandler().addMessage(Level.ERROR, msg);
```
Comment (kevinleturc) [inline modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/snapshot/SnapshotManagerComponent.java:424]:
```suggestion
                addRuntimeMessage(Level.ERROR, msg);
```
?

**Comment by atchertchian** [comment]
Waiting for a nuxeo master valid deployment

**Comment by atchertchian** [comment]
Merging now as this is impacting builds from all branches

**Pull request merged**